server/src/main/java/org/opensearch/action/admin/indices/replication/SegmentReplicationShardStatsResponse.java

```diff
@@ -31,19 +32,31 @@ public class SegmentReplicationShardStatsResponse implements Writeable {
     @Nullable
     private final SegmentReplicationState replicaStats;
 
+    @Nullable
+    private final SegmentReplicationShardStats segmentReplicationShardStats;
```

Comment:
if this only holds data for search replicas can we rename this do indicate that?

Reply:
Sure